scripts/mkcloud

```diff
@@ -151,6 +151,7 @@ function show_environment()
     echo "-------------------------------"
     echo " cloudsource: $cloudsource"
     echo "    TESTHEAD: $TESTHEAD"
+    echo "    scenario: $scenario"
```

Comment:
Thanks, I was missing the context, maybe it should be part of the PR description :)